tests/test_cmd.py

```diff
@@ -16,7 +16,7 @@ class CmdTestCase(unittest.TestCase):
     def test_execute(self, mock_debug):
       err = execute("echo foo", mock_debug)
       self.assertEqual(err, 0)
-      self.assertEqual(mock_debug.mock_calls, [call('foo')])
+      self.assertEqual(mock_debug.mock_calls, [call("%s", "foo")])
```

Comment:
Why the extra formatting?

Reply:
This is changed because of the change above. In the general case, an output line can contain `%`, so this needs the extra `%s`.